src/index.html

```diff
@@ -4,9 +4,74 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="../styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month">
+      <div class="month__day month__day--start-day-wed">
+      </div>
+      <div class=" month__day">
```

Comment:
do not add spaces, keep it straight
```suggestion
      <div class="month__day">
```